### PR TITLE
Export/Import Logins

### LINKED ISSUE #26: Implement Import/Export of logins
Implement a feature where user can import export the logins as a single file, Import will generate a single file and export will take a file and save all logins from that file to DB

### PR DESCRIPTION
**Pull request by KirbyNguyen**
I have implemented a way for the user to export all the logins into a JSON file. The app will also be able to read from this JSON file and add the data back into the database.
Some shortcomings: the user cannot choose the location nor the name of the export and import file. Right now, the file will be defaulted to "loginData.json." Additionally, the import will delete everything in the database before repopulating it. 

P/S: I was working on this one in my fork so there's no branch with the issue name :( sorry

### CODE REVIEW

**Comment by singhsaubhik** [comment]
> I have implemented a way for the user to export all the logins into a JSON file. The app will also be able to read from this JSON file and add the data back into the database.
> Some shortcomings: the user cannot choose the location nor the name of the export and import file. Right now, the file will be defaulted to "loginData.json." Additionally, the import will delete everything in the database before repopulating it.
> 
> P/S: I was working on this one in my fork so there's no branch with the issue name :( sorry

I will look into it, thanks for PR

File: lib/components/local_path.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path_provider/path_provider.dart';
+
+class LocalPath {
+  LocalPath._privateConstructor();
+  static final LocalPath instance = LocalPath._privateConstructor();
```
Comment (singhsaubhik) [inline lib/components/local_path.dart]: Move this to service directory and rename this file as `file_service.dart`

File: lib/components/local_path.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  final String path = "assets";
```
Comment (singhsaubhik) [inline lib/components/local_path.dart]: It seems this is not used anywhere

**Review by singhsaubhik — APPROVED** [review]

**Review by singhsaubhik — CHANGES_REQUESTED** [review]
ARR

File: lib/components/local_path.dart (hunk 2 above)
Comment (KirbyNguyen) [reply lib/components/local_path.dart]: I have removed that from the code

File: lib/components/local_path.dart (hunk 1 above)
Comment (KirbyNguyen) [reply lib/components/local_path.dart]: Just moved it to services

**Comment by KirbyNguyen** [comment]
I don't know if the new commit shows on here, should I make a new pull request?

**Review by singhsaubhik — APPROVED** [review]

**Pull request merged**